<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mwDynEval</class>
 <widget class="QMainWindow" name="mwDynEval">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dyn Eval</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitter_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QWidget" name="">
        <layout class="QVBoxLayout" name="vlSceneNodes">
         <property name="spacing">
          <number>0</number>
         </property>
        </layout>
       </widget>
       <widget class="QWidget" name="">
        <layout class="QVBoxLayout" name="vlNodeOptions">
         <property name="spacing">
          <number>0</number>
         </property>
        </layout>
       </widget>
      </widget>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QWidget" name="">
        <layout class="QVBoxLayout" name="vlCaches">
         <property name="spacing">
          <number>0</number>
         </property>
        </layout>
       </widget>
       <widget class="QWidget" name="">
        <layout class="QVBoxLayout" name="vlCacheInfo">
         <property name="spacing">
          <number>0</number>
         </property>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="mOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QMenu" name="mFilters">
     <property name="tearOffEnabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Filters</string>
     </property>
    </widget>
    <addaction name="miToolTips"/>
    <addaction name="miNamespace"/>
    <addaction name="miRfDisplay"/>
    <addaction name="miRefreshUi"/>
    <addaction name="mFilters"/>
   </widget>
   <widget class="QMenu" name="mFiles">
    <property name="title">
     <string>Files</string>
    </property>
    <widget class="QMenu" name="mOpenRootPath">
     <property name="title">
      <string>Open Root Path</string>
     </property>
     <addaction name="miXplorer"/>
     <addaction name="miXterm"/>
    </widget>
    <addaction name="miSetRootPath"/>
    <addaction name="mOpenRootPath"/>
   </widget>
   <addaction name="mFiles"/>
   <addaction name="mOptions"/>
  </widget>
  <action name="miToolTips">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>ToolTips</string>
   </property>
  </action>
  <action name="actionToto">
   <property name="text">
    <string>toto</string>
   </property>
  </action>
  <action name="miRefreshUi">
   <property name="text">
    <string>Refresh Ui</string>
   </property>
  </action>
  <action name="miSetRootPath">
   <property name="text">
    <string>Set Root Path</string>
   </property>
  </action>
  <action name="miXplorer">
   <property name="text">
    <string>Xplorer</string>
   </property>
  </action>
  <action name="miXterm">
   <property name="text">
    <string>Xterm</string>
   </property>
  </action>
  <action name="miNamespace">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Namespace</string>
   </property>
  </action>
  <action name="miRfDisplay">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Refresh Display</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
